script.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function(){
+	
+	console.log("get document ready");
+
+	$(".readmore").click(slideDown);
+	
+	$(".readless").click(slideUp);
+
+	$(".learnmore").click(showMore);
+
+	function slideDown() {
+		$("#show-this-on-click").show();
+		$(".readmore").hide();
+		$(".readless").show();
+	}
```

Comment:
You are missing the `event.preventDefault` which tells those `a` links to not do their intended function, which is to reload the page. If the page reloads, your JS never gets a chance to run with the `.click();` event.